AZ3166/src/variants/MXChip_AZ3166/variant.cpp

```diff
@@ -87,7 +87,8 @@ extern const PinDescription g_APinDescription[]=
 /*
  * UART objects
  */
-UARTClass Serial;
+UARTClass Serial(0);
+UARTClass Serial1(1);
```

Comment:
don't have the instantiation object for UART 1, which will increase the memory consumption.
people can instante it when they need.

Reply:
Oh I see I leave the `extern UARTClass Serial1` in variant.h, and then just add `UARTClass Serial1(1)` with my globals.  Is that how you were thinking it should work?

Reply:
The UARTClass herder file is already in Arduino.h file, so don't need to import again.
You can just instantiate it in your sketch as a static variable.